solr/core/src/java/org/apache/solr/handler/BlobHandler.java

```diff
@@ -22,6 +22,7 @@
 import static org.apache.solr.common.params.CommonParams.SORT;
 import static org.apache.solr.common.params.CommonParams.VERSION;
 
+import com.google.common.collect.Lists;
```

Comment:
isn't there a PR to get away from this?

Reply:
Mm, that's news to me, I'll keep an eye out and stop using it if it goes through.

Reply:
never mind, I dug around in JIRA and can't find it!